app/task/Cache.php

```diff
@@ -40,26 +43,29 @@ class Cache extends AbstractTask
      */
     public function clear()
     {
-        $this->output('Start');
+        (new CLImate())->out('Start');
```

Comment:
could you improve base `output` method instead to use `(new CLImate())` each time